code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -304,19 +304,19 @@
 		if(op != "")
 			switch(op)
 				if("+")
-					result += val
+					result = (result + val)
```

Comment:
>How many god damn bugs have we had this last year from non-changes like that. I'm sick of them.
 
And if  you left the fucking comment, guess how often we'd have to deal with this in the future? 0